src/tailwind.js

```diff
@@ -118,6 +118,7 @@ module.exports = {
     // https://v1.tailwindcss.com/docs/configuring-variants#ordering-variants
     backgroundColor: ["responsive", "hover", "focus", "active"],
     textColor: ["responsive", "hover", "focus", "active"],
+    borderRadius: ["responsive", "hover", "focus", "active"],
```

Comment:
y do we need this here, by default is the border radius not inside the hover state?

Reply:
ya by default the borderRadius is not inside the hover state.

Reply:
ok